### ISSUE TITLE
[PapyrusDotNet] Add a argument -x for automatically exit when the process is done.

### ISSUE THREAD

**Issue by zerratar** [zerratar/PapyrusDotNet#8]
Currently, the UI will need you to press on X (on the keyboard) for closing (or by alt+f4, ctrl+c or clicking the X button) when the process is completed. This should be optional as PapyrusDotNet will most likely be used as a post build process.

**Comment by zerratar** [comment]
Fixed in commit: 4747d411ade98eb0e82753510e31003667ba20de

**Comment by Outpox** [comment]
Hey there seems to still be some issue when adding -x
When executing the command line in a cmd it works as it should (closes automaticaly) : 
```
C:\Users\Outpox\Documents\PapyrusDotNet\Binaries\PapyrusDotNet.exe C:\Users\Outpox\Documents\TAP_Script\TAP_Script\bin\Debug\TAP_Script.dll "B:\SteamLibrary\steamapps\common\Fallout 4\Data\Scripts" -x
```

![image](https://cloud.githubusercontent.com/assets/3439246/15451174/7bedf188-1fb5-11e6-88a5-fc984cf8174f.png)

But when executing my `.bat` file as a post-build process in VS 2015 Community it throw an error and the file doesn't compile. 

![image](https://cloud.githubusercontent.com/assets/3439246/15451179/af8a3a2e-1fb5-11e6-9c76-5fcf62d7b7cb.png)

![image](https://cloud.githubusercontent.com/assets/3439246/15451182/bdcde8ba-1fb5-11e6-98eb-2105c060da4b.png)

Seems like the since there's no window from the post-build execution, `Move()` and `DrawRectangle()` and `DrawInterface()` are not available.  
  
Will create a pull request if I manage to edit this project sources :monkey_face: (got references issues that I need to figure out)
